packages/framework/src/Model/Feed/FeedExportFactory.php

```diff
@@ -10,11 +10,6 @@
 
 class FeedExportFactory
 {
-    /** @access protected */
```

Comment:
ok, makes sense as they are not used anywhere. However, I would expect this in another commit

Reply:
I have added info into commit message